docs/whats_new.rst

```diff
@@ -5,23 +5,40 @@
 
 .. _whats_new:
 
-Changelog
-============
+The NextCord Update Changelog
+==============================
 
 This page keeps a detailed human-friendly rendering of what's new and changed
 in specific versions.
 
-.. _vp2p0p0:
-
 v2.0.0
 --------
+Warning: **This Does Not Show The Changes In discord.py V2**
```

Comment:
Maybe add a link for where to find them?
Also shouldn't warning also be bold

Reply:
I don't currently think there is a list of changes